docs/data/data-grid/layout/layout.md

```diff
@@ -3,15 +3,13 @@
 <p class="description">The data grid offers multiple layout modes.</p>
 
 :::error
-By default, the data grid has **no intrinsic dimensions**. It occupies the space its parent leaves.
-The grid will raise an error in the console if its container has no intrinsic dimensions.
+By default, the data grid has **no intrinsic dimensions**.
```

Comment:
Always capitalize "Data Grid" whenever you see it, to reinforce the product name vs. the general concept of the thing. See: https://www.notion.so/mui-org/Style-conventions-for-MUI-components-and-products-dff7d2d7d1ba4a4abd47f48cf345b800

```suggestion
By default, the Data Grid has **no intrinsic dimensions**.
```

Reply:
Ahh thanks for the reminder! I made a similar change for "Date Picker"!